File: src/backend/TaskControlApp/TaskControlApp.Domain/Entities/TaskItem.cs
namespace TaskControlApp.Domain.Entities
{
    public class TaskItem
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Created { get; set; }
        public string Updated { get; set; }
        public int Priority { get; set; }
    }
}

File: src/frontend/task-control-app/src/core/TaskItem.ts
export default class TaskItem {
    id: number
    name: string
    description: string
    created: string
    updated: string
    priority: number

    constructor(id: number = 0 ,name: string = '', description: string = '', created: string, updated: string, priority: number = 0) {
        this.id = id
        this.name = name
        this.description = description
        this.created = created
        this.updated = updated
        this.priority = priority
    }
}